File: RockPaperScissors.java
import java.util.Random;
import java.util.Scanner;

public class RockPaperScissors {

	private static String rock = "🪨(Rock)";
	private static String paper = "📰(Paper)";
	private static String scissors = "✂(Scissors)";
	
	public static void play() {
		Scanner scan = new Scanner(System.in);
		Random rand = new Random();
		
		System.out.print("What is your choice? \n'R' for " + rock + ", 'P' for " + paper + ", 'S' for " + scissors + "\n>> ");
		String user = scan.next();
		user = user.toUpperCase();
		
		String[] comp_list = {"R", "P", "S"};
		String computer = comp_list[rand.nextInt(3)];
		
		if (user.equals(computer)) {
			System.out.println("It is a tie. You and the computer have both chosen " + return_word(user) + ".");
		}
		else {
			if ((user.equals("R") && computer.equals("S")) || (user.equals("P") && computer.equals("R")) || (user.equals("S") && computer.equals("P"))) {
				System.out.println("You chose " + return_word(user) + " and the computer chose " + return_word(computer) + ".\nYou won!");
			}
			else {
				System.out.println("You chose " + return_word(user) + " and the computer chose " + return_word(computer) + ".\nYou lose :(");
			}
		}
	}
	
	public static String return_word(String choice) {
		
		if (choice.equals("R")) {
			return rock;
		}
		else if (choice.equals("P")) {
			return paper;
		}
		else if (choice.equals("S")) {
			return scissors;
		}
		
		return "";
		
	}
	
	public static void main(String[] args) {
		play();
	}

}

File: RockPaperScissors.py
import random

rock = '\U0001faa8' + "(Rock)"
paper = '\U0001f4c3' + "(Paper)"
scissors = '\u2702' + "(Scissors)"

def play():

    # get user's input
    user = input("What is your choice? \n'R' for " + rock + ", 'P' for " + paper + ", 'S' for " + scissors + "\n>> ")
    # convert user's input to uppercase
    user = user.upper()

    # get computer's choice
    choice_list = ['R', 'P', 'S']
    computer = random.choice(choice_list)

    # logic statement
    # if both inputs are the same, it is a tie
    if user == computer:
        print('It is a tie. You and the computer have both chosen', return_word(user), '.')

    # if both inputs are different, check whether the user beats the computer
    else:
        # winning conditions: R > S, S > P, P > R
        if (user == 'R' and computer == 'S') or (user == 'S' and computer == 'P') or (user == 'P' and computer == 'R'):
          # print user's win statement
          print('You chose', return_word(user), 'and the computer chose', return_word(computer), '.\nYou won!')
        # losing conditions: other than winning conditions
        else:
          # print user's lose statement
          print('You chose', return_word(user), 'and the computer chose', return_word(computer), '.\nYou lose :(')

def return_word(choice):
    if choice == 'R':
        return rock
    elif choice == 'P':
        return paper
    elif choice == 'S':
        return scissors

play()
while input("Play Again? (Y/N) ").upper() == "Y":
    play()